backend/src/schema/resolvers/passwordReset.spec.js

```diff
@@ -15,10 +15,13 @@ let variables
 
 const getAllPasswordResets = async () => {
   const session = driver.session()
-  const transactionRes = await session.run('MATCH (r:PasswordReset) RETURN r')
-  const resets = transactionRes.records.map(record => record.get('r'))
-  session.close()
-  return resets
+  try {
+    const transactionRes = await session.run('MATCH (r:PasswordReset) RETURN r')
+    const resets = transactionRes.records.map(record => record.get('r'))
+    return resets
+  } finally {
+    session.close()
+  }
```

Comment:
why aren't you using `neode.cypher` here?

Reply:
:man_shrugging: it's a test so I wouldn't mind using `neode` here. But using plain javascript driver is not a problem either, oder?

Reply:
I don't thinks it's a problem, but it's more code, and I'm curious to know how `neode.cypher` handles promises/closing sessions, etc...